outofcore/include/pcl/outofcore/octree_base.h

```diff
@@ -619,9 +619,8 @@ namespace pcl
         void
         flushToDiskLazy ();
 
-        /** \brief DEPRECATED - Flush empty nodes only 
-         *  \deprecated
-         */
+        /** \brief Flush empty nodes only */
+        PCL_DEPRECATED(1, 13, "Use DeAllocEmptyNodeCache(OutofcoreNodeType* current) with no arguments instead")
```

Comment:
Following the instruction we provide in this message will still trigger the deprecation warning, because the overload resolution will still call `DeAllocEmptyNodeCache ()` if no arguments are passed to the method. Better to just flag the ABI break and delete the method.

Reply:
Or remove the deprecation and keep the method (both work)

Reply:
True. I have no issues in keeping it around.

Reply:
Ah you're right. You really prefer keeping the duplicate method?

Reply:
@kunaltyagi is right. I personally don't really see a reason to deprecate it so maybe we shouldn't. We were just following a cue from some past developer. If it ain't broken, no need to fix it :)